scripts/common.sh.in

```diff
@@ -353,18 +353,13 @@ configure_salt_minion_local_mode() {
         pillar="{'metalk8s': {'archives': '$BASE_DIR'}}" saltenv=base
 }
 
-check_local_node() {
-    # NOTE: Today using bootstrap script, bootstrap node is also the CA
-    # minion, so check for minion id equal to CA minion
-    minion_id="$("$SALT_CALL" --out txt --local grains.get id | cut -c 8-)"
-    ca_minion="$("$SALT_CALL" --out txt --local pillar.get metalk8s:ca:minion | cut -c 8-)"
-    if [ "$minion_id" != "$ca_minion" ]; then
-        echo "CA minion \"$ca_minion\" from bootstrap configuration is not equal" \
-            "to the local minion ID \"$minion_id\", you need to change" \
-            "the local minion ID, or update the bootstrap configuration." 1>&2
-        return 1
-    fi
+ensure_archives_mounted() {
+    "${SALT_CALL}" --local --state-output=mixed --retcode-passthrough \
+        state.sls metalk8s.archives.mounted \
+        saltenv=metalk8s-@@VERSION
```

Comment:
We don't need `pillarenv=metalk8s-@@VERSION` anymore ?

Reply:
For me, It was never really necessary

Reply:
For me neither, but I was just wondering if it was on purpose.